content/deploy/community-cloud/_index.md

```diff
@@ -5,11 +5,11 @@ slug: /deploy/streamlit-community-cloud
 
 # Welcome to Streamlit Community Cloud
 
-Deploy, manage, and share your Streamlit apps with the world in minutes &mdash; all for free. Your Streamlit Community Cloud account connects directly to your GitHub repository (public or private). Most apps will launch in only a few minutes. Streamlit Community Cloud handles all of the containerization so all you need to worry about is creating the app. Rapidly prototype, explore, and update apps by simply changing your code in GitHub. Most changes appear immediately!
+With Streamlit Commuity Cloud, you can create, deploy, and manage your Streamlit apps &mdash; all for free. Share your apps with the world and build a customized profile page to show off your work. Your Streamlit Community Cloud account connects directly to your GitHub repositories (public or private). Most apps will launch in only a few minutes. We handle all of the containerization, so deploying is easy. Bring your own code, or start from one of our popular templates. Rapidly prototype, explore, and update apps by simply changing your code in GitHub. Most changes appear immediately!
```

Comment:
"We" are the people who create and maintain the Streamlit library and Streamlit Community Cloud. "We" is used through the docs, so it might be a larger discussion to remove it more broadly.